maintainers/maintainer-list.nix

```diff
@@ -14669,4 +14669,10 @@
     github = "npatsakula";
     githubId = 23001619;
   };
+  _0xC45 = {
+    email = "jason@0xc45.com";
+    name = "Jason Vigil";
+    github = "0xC45";
+    githubId = 56617252;
+  };
```

Comment:
this should be added in a separate commit (`maintainers: add 0xC45`) and added to the top of the file in the properly sorted place